libs/blocks/quiz/quiz.css

```diff
@@ -1,980 +1,384 @@
-.quiz { 
-  min-height: 1000px;
-  display: flex;
-  align-items: center;
-  flex-direction: column;
+.quiz-page * {
```

Comment:
`*` does not account for `:before` and `:after` pseudoelements, do they need to be included?

Reply:
No, this is not needed for pseudoelements, which only appear on the progress indicators.